02_julia_intro/02_julia_intro.jl

```diff
@@ -516,10 +516,135 @@ julia> map(x -> x^2 + 5, [2, 4, 6, 3, 3])
  14
 ```
 The first argument of the map function, is another function. You can use functions already defined if you want, but with the help of anonymous functions you can simply create a function that will be used only inside map. The function we specified in the first argument, is then applied to every member of the array in the second argument. 
-
-Now that the most important details of syntax have been discussed, let's focus our attention into some packages of Julia's ecosystem.
 "
 
+# ╔═╡ fda97ed8-875f-11eb-372e-4f982ff9b958
+md"""Now lets introdice another data collection: Dictionaries.
+A dictionary is a collection of key-value pairs.
+You can think them like as an array but insted of being indexed by a sequence of numbers they are indexed based on keys, where each key connects to their value.
```

Comment:
You can think of them as an array, but insted of being indexed by a sequence of numbers they are indexed by keys, each one linked to a value.